src/web/web.cpp

```diff
@@ -241,7 +241,85 @@ nlohmann::json web::context::post(Endpoint ep, const std::string& specifier, con
 
     return qb::json_utils::parse_safe(res.body());
 }
+nlohmann::json web::context::patch(const std::string& uri, const std::string& body)
+{
+    assert(initialized_);
+
+    qb::log::scope<std::string> slg("[HTTP PUT request creation start.]");
```

Comment:
very small thing but this should probably say `PATCH`
but this will be fixed anyways when this entire file is rewritten so, shrug